00_SETUP.md

```diff
@@ -114,6 +114,31 @@ From the directory containing the notebooks:
 ```console
 jupyter lab
 ```
+
+## 8. Handling Notebooks in MyST-Markdown format
+
+The Jupyter notebooks in this repository are in
+[MyST-Markdown format](https://myst-nb.readthedocs.io/en/v0.13.2/use/markdown.html).
+The [jupytext](https://jupytext.readthedocs.io/en/latest/index.html) package is
+included in your `navo-env` environment to work with these notebooks. Note that
+the `jupytext` package has to be installed before your Jupyterlab session starts.
+
+To open one of these notebooks (in the `content/reference_notebooks` and the
+`content/use_case_notebooks` subdirectories), in the Jupyterlab file panel,
+right-click on the notebook, choose "Open With", and select "Notebook" or
+"Jupytext Notebook" from the drop-down menu.
```

Comment:
maybe add a screenshot here, too? (only because there is a screenshot for the more complicated case).

Reply:
Added the screenshot.

Reply:
Thanks.